scripts/FRI_soundness_list_decoding.sage

```diff
@@ -0,0 +1,286 @@
+from math import log2
+
+
+def commit_soundness_error_list(field_bits, lde_domain_bits, trace_domain_bits, fold_arities, num_polys, m, quotient_degree=0):
+    # Returns the log2 soundness error of the FRI commit phase
```

Comment:
Maybe worth to point out that this is for "algebraic" batching. Is there a specific reason why `_list()` is in the name of many functions?

Reply:
We have a remark related to `num_polys` to cover both algebraic as well as affine batching. (`num_polys: The degree of the batching plus 1 e.g. for linear batching it is equal to 2.`)
The use of `_list()` is to emphasis that these functions are in the list-decoding regime. Hopefully, in the near future, we can add similar functions for the unique decoding regime.